user_test.go

```diff
@@ -5,12 +5,40 @@ import "testing"
 func TestUser(t *testing.T) {
 	t.Parallel()
 
-	user := &User{
-		Username:      "bob",
-		Discriminator: "8192",
+	tests := []struct {
+		name string
+		u    *User
+		want string
+	}{
+		{
+			name: "Given a user with a username and discriminator, When String() is called, Then it should return the username and discriminator",
```

Comment:
I see, so the explanation is just about the action without the need to describe the cause and the result. I think it will better if u have the standardization about the naming tests for the future.

Reply:
I think the simplify is better, I'll commit to this suggestion